textattack/shared/attacked_text.py

```diff
@@ -64,10 +83,10 @@ def __hash__(self):
     def free_memory(self):
         """ Delete items that take up memory.
             
-            Can be called once the TokenizedText is only needed to display.
+            Can be called once the AttackedText is only needed to display.
         """
-        self.ids = None
-        self.tokenizer = None
+        if "previous_attacked_text" in self.attack_attrs:
+            self.attack_attrs["previous_attacked_text"].free_memory()
```

Comment:
Well it's just a pointer to the previous text, so I think it makes more sense to keep it. My idea was because I thought there should be a format where we print all the steps of a perturbation from the original to perturbed

Reply:
Hmm yeah that might be useful for the demo.